doap.rdf

```diff
@@ -43,8 +43,8 @@
     <release>
       <Version>
         <name>1.10.0</name>
-        <created>2025-09-11</created>
-        <revision>1.10.0</revision>
+        <created>2025-12-22</created>
+        <revision>1.10.1</revision>
```

Comment:
The past release commits have changed the above `name` as well. Why do we skip updating it in this PR? 

* https://github.com/apache/iceberg/commit/026374b530eeea46429cef61a99ba9321e0587b6
* https://github.com/apache/iceberg/commit/8eb476f4b8cd918ec62ec3470b47d6da3204fd1c

Reply:
Fixed